www/include/configuration/configObject/service/listServiceByHost.php

```diff
@@ -84,15 +84,15 @@
     $centreon->historySearch[$url]["searchS"] = $searchS;
     isset($_POST["statusHostFilter"]) ? $hostStatus = 1 : $hostStatus = 0;
     $centreon->historySearch[$url]["hostStatus"] = $hostStatus;
-} elseif (isset($_GET['Search'])) {
+} elseif (isset($_GET['search'])) {
     $centreon->historySearch[$url] = array();
-    $template = $_GET['template'];
+    $template = $_GET['template'] ?? '';
     $centreon->historySearch[$url]['template'] = $template;
-    $status = $_GET["status"];
+    $status = $_GET["status"] ?? '';
     $centreon->historySearch[$url]["status"] = $status;
     $searchH = $_GET["searchH"];
     $centreon->historySearch[$url]["searchH"] = $searchH;
-    $searchS = $_GET["searchS"];
+    $searchS = $_GET["searchS"] ?? '';
```

Comment:
searchS was initialized to null on line 66

Reply:
the old code was $searchS = $_GET["searchS"] which produced PHP warning. Fixed now.